src/node.cc

```diff
@@ -3595,6 +3595,8 @@ static void CheckIfAllowedInEnv(const char* exe, bool is_env,
     "--force-fips",
     "--openssl-config",
     "--icu-data-dir",
+    "--perf-prof",
+    "--perf-basic-prof",
```

Comment:
Move them below the comment and use underscores.

Reply:
@bnoordhuis Thank you for review !
I moved them under v8 options.